aztarna/ros/ros/scanner.py

```diff
@@ -44,7 +43,7 @@ def __init__(self):
             ros_host = ROSHost(address, port)
             async with self.semaphore:
                 try:
-                    code, msg, val = await ros_master_client.getSystemState('')
+                    code, val = await ros_master_client.getSystemState('')
```

Comment:
I think this is wrong.

Reply:
The warning was, unused variable. But, if should be like this I'm going to revert the change.